src/data/abilities/ab-attrs/post-battle-loot-ab-attr.ts

```diff
@@ -2,28 +2,42 @@ import { PostBattleAbAttr } from "#abilities/post-battle-ab-attr";
 import { globalScene } from "#app/global-scene";
 import { getPokemonNameWithAffix } from "#app/messages";
 import type { Pokemon } from "#field/pokemon";
+import type { PokemonHeldItemModifier } from "#modifier/modifier";
 import { randSeedItem } from "#utils/random-utils";
 import i18next from "i18next";
 
 export class PostBattleLootAbAttr extends PostBattleAbAttr {
-  public override apply(pokemon: Pokemon, simulated: boolean, isVictory: boolean): boolean {
+  private randItem?: PokemonHeldItemModifier;
+
+  public override apply(pokemon: Pokemon, simulated: boolean, _isVictory: boolean): void {
     const postBattleLoot = globalScene.currentBattle.postBattleLoot;
```

Comment:
Technically this `AbAttr` doesn't need to check `simulated` because it should never be `true` for this.
```suggestion
    if (simulated) {
      return;
    }
    const postBattleLoot = globalScene.currentBattle.postBattleLoot;
```